crates/webidl/src/util.rs

```diff
@@ -552,107 +687,107 @@ impl TypeToString for weedle::types::StringType {
     }
 }
 
-impl TypeToString for weedle::term::Byte {
-    fn type_to_string(&self, _record: &FirstPassRecord, dst: &mut String) {
+impl GetTypeName for weedle::term::Byte {
```

Comment:
Either in this PR or in a follow up, I think it would be great to write a macro that cleans up all these trivial implementations, so we could just do:

```rust
simple_type_names! {
    weedle::term::Byte => "i8",
    weedle::term::Octet => "u8",
    // etc...
}
```